pkg/backend/openshiftcluster/create.go

```diff
@@ -269,6 +271,9 @@ func (m *Manager) Create(ctx context.Context) error {
 					ControlPlaneSubnet:       masterSubnetName,
 					ComputeSubnet:            workerSubnetName,
 					ARO:                      true,
+					SubscriptionID:           r.SubscriptionID,
```

Comment:
What if we add those into `bootstraplogging.Credentials` or bootstraplogging.Config` new asset so removing blast radius of the change a bit? We are already passing in credentials structure to template as asset. It can have all the required fields in it?